packages/generator-helper/src/generatorHandler.ts

```diff
@@ -24,8 +24,10 @@ export function generatorHandler(handler: Handler): void {
           jsonrpc: '2.0',
           error: {
             code: -32000,
-            message: e.stack || e.message,
-            data: null,
+            message: e.message,
```

Comment:
This is needed for separating stack-trace and the error message like with normal js errors. Previous code put stacktrace into the message itself.